seed-db.ts

```diff
@@ -757,8 +757,9 @@ void (async function setupDevDB() {
             stepName: 'Verifizieren',
             type: achievement_type_enum.SEQUENTIAL,
             subtitle: 'Jetzt durchstarten',
-            description: 'Dieser Text muss noch geliefert werden.',
-            image: 'gamification/achievements/tmp/finish_onboarding/four_pieces/empty_state.png',
+            description:
+                'Hurra! Am {{date}} haben wir eine E-Mail an deine Adresse {{email}} gesendet. Um deine E-Mail zu bestätigen, klicke einfach auf den Button in der Nachricht. Solltest du unsere E-Mail nicht finden, kannst du hier eine erneute Zustellung anfordern und voller Vorfreude auf unser Weiterkommen warten.',
```

Comment:
We should also add the date on which the verification message was sent - and the email - to the context. We get this information from `concreteNotification.createdAt`

Reply:
I used the varifiedAt value inside the function, called by the mutation.